cmd/kube-apiserver/app/options/options_test.go

```diff
@@ -80,7 +80,7 @@ func TestAddFlags(t *testing.T) {
 		"--audit-webhook-truncate-max-batch-size=43",
 		"--audit-webhook-truncate-max-event-size=42",
 		"--audit-webhook-initial-backoff=2s",
-		"--audit-webhook-version=audit.k8s.io/v1alpha1",
```

Comment:
revert this, this is testing that explicitly setting the flag can override the default